rush.json

```diff
@@ -1,11 +1,14 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.34.2",
-  "pnpmVersion": "4.12.5",
-  "nodeSupportedVersionRange": ">=10.17.0 <15.0.0",
+  "rushVersion": "5.47.0",
+  "pnpmVersion": "6.4.0",
+  "nodeSupportedVersionRange": ">=12.17.0 <15.0.0",
```

Comment:
Why did this need to be updated to 12?

Reply:
Per change log, they no longer support node v10. (`Node.js v10 support is dropped. At least Node.js v12.17 is required for the package to work.`) https://github.com/pnpm/pnpm/releases/tag/v6.0.0

Reply:
Okay, we'll need to add that to the base README for preferred tools.